tests/scripts/pulp_deb/test_deb_remote.sh

```diff
@@ -13,10 +13,10 @@ trap cleanup EXIT
 # Fail to create some remotes:
 expect_fail pulp deb remote create --name "foo" --url "foo" --distribution ""
 expect_fail pulp deb remote create --name "foo" --url "foo"
-expect_fail pulp deb remote create --name "foo" --url ""    --distribution "foo"
-expect_fail pulp deb remote create --name "foo"             --distribution "foo"
-expect_fail pulp deb remote create --name ""    --url "foo" --distribution "foo"
-expect_fail pulp deb remote create              --url "foo" --distribution "foo"
+expect_fail pulp deb remote create --name "foo" --url "" --distribution "foo"
+expect_fail pulp deb remote create --name "foo" --distribution "foo"
+expect_fail pulp deb remote create --name "" --url "foo" --distribution "foo"
+expect_fail pulp deb remote create --url "foo" --distribution "foo"
```

Comment:
I don't have a strong opinion, but I would like a consensus going forward if we want to format these test scripts in a manual block aligned style, or if we want to go ahead with these formatting changes?